sci-libs/spqr/spqr-2.0.9-r4.ebuild

```diff
@@ -30,6 +30,9 @@ RDEPEND="${DEPEND}"
 src_prepare() {
 	default
 	elibtoolize
+
+	sed -e '/^\\usepackage{hyperref}$/i \\\\usepackage{html}' \
+		-i Doc/spqr_user_guide.tex || die
```

Comment:
Thanks. I think this would be nicer as a `PATCHES=( ... )` instead of running `sed`. (See e.g. https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/)

Reply:
this problem had beed fixed in upstream (by remove \html), so I choose sed for a quick fix.